### ISSUE TITLE
[xcattest] there are illegal colon used in testcase

### ISSUE THREAD

**Issue by robin2008** [xcat2/xcat-core#3575]
I found something wrong when I run xcattest, for example

```
xcattest -f abc.conf -t mkdef_node_with_a_node_template
...
RUN: [Mon Jul 31 21:49:34 2017]
mkdef -t node -o auto_test_cec_node_1 --template cec-template serial=test mtm=test hcp=test groups=test_template
check：rc==0
ElapsedTime:1 sec
RETURN rc = 127
OUTPUT:
1 object definitions have been created or modified.
/tmp/xCATautotest1501552174//script: line 2: $'check\357\274\232rc==0': command not found
CHECK:output =~ 1 object definitions have been created or modified	[Pass]
CHECK:output =~ created	[Pass]
```

Check the codes, it is caused by the wrong colon,  and also found the same issue in other cases. It is better to check all of them.
We need grep all case

**Comment by zet809** [comment]
hi, @hu-weihua , is this still a problem?

**Comment by hu-weihua** [comment]
@robin2008 @zet809 , this issue has been fixed by #4911. could you help to verify and close this issue? thanks
